### PR TITLE
[Behat] Rework behat - using attributes instead class/id part in summary page

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | based on  #10940
| License         | MIT

rework summary page

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/Complete/_form.html.twig [hunk 1]
```diff
@@ -1 +1 @@
-{{ form_row(form.notes, {'attr': {'rows': 3}}) }}
+{{ form_row(form.notes, sylius_test_form_attribute('extra-notes')|merge({'rows': 3})) }}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/Complete/_form.html.twig]: Shouldn't it be
```suggestion
{{ form_row(form.notes, sylius_test_form_attribute('extra-notes')|sylius_merge_recursive({'attr': {'rows': 3}})) }}
```
?

File: src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Table/_totals.html.twig [hunk 2]
```diff
@@ -34,11 +34,13 @@
...
 </tr>
 <tr>
-    <td colspan="4" id="promotion-total" class="right aligned">
+    <td colspan="4" id="promotion-total" class="right aligned" {{ sylius_test_html_attribute('promotion-total') }}>
         {{ 'sylius.ui.discount'|trans }}: {{ money.convertAndFormat(order.orderPromotionTotal)  }}
         {% if order.orderPromotionTotal != 0 %}
             <i id="order-promotions-details" class="question circle icon popup-js"
-                data-html="{% for key, value in orderPromotions %}<div>{{ key }}: {{ money.convertAndFormat(value) }}</div>{% endfor %}">
+                data-html="{% for key, value in orderPromotions %}<div>{{ key }}: {{ money.convertAndFormat(value) }}</div>{% endfor %}"
+
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Table/_totals.html.twig]:
```suggestion
```

**Comment by Tomanhez** [comment]
Replaced by : https://github.com/Sylius/Sylius/pull/11032

**Comment by AdamKasp** [comment]
replaced by #11032

**Pull request closed without merging**